tachyon/zk/plonk/examples/tracking_testing.md

```diff
@@ -24,6 +30,8 @@ The following circuits result in the same test data when using `SimpleFloorPlann
 
 - SimpleLookupCircuit
 - ShuffleCircuit
+- Fibonacci2Circuit
```

Comment:
```suggestion
- Fibonacci1Circuit
- Fibonacci2Circuit
```

Reply:
Fibonacci1 has values that change when using v1/simple floor planner